frontends/web/src/routes/account/send/send.tsx

```diff
@@ -154,13 +213,15 @@ export default class Send extends Component {
                     data: null,
                 });
                 if (this.utxos) {
+                    // @ts-ignore
```

Comment:
I would prefer if you cast the `getWrappedInstance()` (Which I guess is necessary? Or what is its type?). After you made `clear()` public, shouldn't the following (or at least something like it) work?

`(this.utxos.getWrappedInstance() as UTXOs).clear();`

This would have the advantage that TypeScript would help us detect if the method changed (its name or arguments) or is no longer there.